src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -119,6 +244,7 @@ export default class VizShonenJump extends Connector {
     async _handleConnectorURI(payload) {
         try {
             // API request
+            const testOrigin = new URL(this.url).origin
```

Comment:
why?

Reply:
Superfluous, again for testing, will remove, thanks!